### PR TITLE
netrc and fetchmailrc docs

### PR DESCRIPTION
**Pull request by h00die**
Adds docs for the post modules `netrc` and `fetchmailrc`.

## Verification

- [x] Start `msfconsole`
- [x] `use netrc`
- [x] `info -d`
- [x] `use fetchmailrc`
- [x] `info -d`
- [x] **Verify** the docs load, things are spelled right etc
- [x] **Verify** the thing does not do what it should not

### CODE REVIEW

File: documentation/modules/post/multi/gather/netrc_creds.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+## Verification Steps
+
+1. Start msfconsole
+1. Get a shell on a system
+1. Do: `use post/multi/gather/netrc_creds`
+1. Do: `set session [session]`
+1. Do: `run`
+1. If any `.netrc` files exist with credentials, they will be stored and read
```
Comment (gwillcox-r7) [inline documentation/modules/post/multi/gather/netrc_creds.md:28]:
```suggestion
1. If any `.netrc` files exist with credentials, they will be read and stored into a loot file.
```

File: documentation/modules/post/multi/gather/fetchmailrc_creds.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+This file is kept in user's home directories to configure fetchmail,
+but contains cleartext credentials.
+
+### Example fetchmailrc file
+
+Example documentation can be found in the fetchmail handbook:
+https://docs.freebsd.org/doc/6.0-RELEASE/usr/share/doc/handbook/mail-fetchmail.html#:~:text=fetchmailrc%20serves%20as%20an%20example,user%20on%20the%20local%20system.
+
+```
+echo "poll example.com protocol pop3 username \"joesoap\" password \"XXX\"" > ~/.fetchamilrc
```
Comment (gwillcox-r7) [inline documentation/modules/post/multi/gather/fetchmailrc_creds.md:15]:
```suggestion
echo "poll example.com protocol pop3 username \"joesoap\" password \"XXX\"" > ~/.fetchmailrc
```

**Review by gwillcox-r7 — CHANGES_REQUESTED** [review]

File: documentation/modules/post/multi/gather/fetchmailrc_creds.md (hunk 2 above)
Comment (gwillcox-r7) [reply documentation/modules/post/multi/gather/fetchmailrc_creds.md]: Fixed in c587360e905a01377cbbbbd7a82571a687cb1200

File: documentation/modules/post/multi/gather/fetchmailrc_creds.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+## Verification Steps
+
+1. Start msfconsole
+1. Get a shell on a system
+1. Do: `use post/multi/gather/fetchmailrc_creds`
+1. Do: `set session [session]`
+1. Do: `run`
+1. If any `.fetchmailrc` files exist with credentials, they will be stored and read
```
[the remark this answers is not in the document]
Comment (gwillcox-r7) [reply documentation/modules/post/multi/gather/fetchmailrc_creds.md]: Fixed in c587360e905a01377cbbbbd7a82571a687cb1200

**Review by gwillcox-r7 — APPROVED** [review]

**Comment by h00die** [comment]
thanks @gwillcox-r7 for the changes

**Pull request merged**